build/gulp.package.js

```diff
@@ -77,7 +77,7 @@ const zipArchive = async ({ osName, binding, tempBin, binaryName }) => {
 }
 
 const makeTempPackage = () => {
-  const additionalPackageJson = require(path.resolve(__dirname, './package.pkg.json'))
+  const additionalPackageJson = require(path.resolve(__dirname, './package.json'))
```

Comment:
Renaming this package from `package.pkg.json` to `package.json` allows me to run the `yarn version --new-version` directly instead of having to call a NodeJS script that updates it manually.

Reply:
How about we just move the content of `package.pkg.json` to the `packages/bp/package.json` file instead ? I don't remember why it was split, but in every other project we added those config directly to the package file

Reply:
Good point, I don't see why I wouldn't work!

Reply:
Here is the PR in question: https://github.com/botpress/botpress/pull/11213